### ISSUE TITLE
Incorrect casing on Typescript imports

### ISSUE THREAD

**Issue by twastvedt** [sequelize/sequelize-auto#483]
When I create a Typescript model, imports in table files have the wrong case. Here's an example:

**DocumentParameter.ts** (this looks ok)
```
...
export type DocumentParameterPk = "DocumentParameterId";
export type DocumentParameterId = DocumentParameter[DocumentParameterPk];
export type DocumentParameterCreationAttributes = Optional<DocumentParameterAttributes, DocumentParameterPk>;

export class DocumentParameter extends Model<DocumentParameterAttributes, DocumentParameterCreationAttributes> implements DocumentParameterAttributes {
...
```

**Document.ts** (Note the casing on the imports and the filename, which don't match above.)
```
...
import type { documentparameter, documentparameterId } from './Documentparameter';
...
```

**init-models.ts** (this looks ok)
```
...
import { DocumentParameter } from "./DocumentParameter";
...
```

The above is with `--caseModel o --caseFile o`. If I change that to `--caseModel p --caseFile p`, the file names stay the same, but `documentparameter` is replaced with `Documentparameter` in `Document.ts`. That's closer, but still missing the capital P.

**Comment by twastvedt** [comment]
Looks like this probably has something to do with my database being on Windows. When run against a database on Linux, `lower_case_table_names = 0`, everything looks ok. On Windows I have `lower_case_table_names` set to 2 and am seeing the above.

**Comment by steveschmitt** [comment]
Which database dialect are you using?  What are your table names?

Thanks for reporting this.

**Comment by chaoren1641** [comment]
Can the class name begin with a larger name? It is now lowercase.

**Comment by steveschmitt** [comment]
@chaoren1641 Use parameter `--caseModel p` to make it use PascalCase.

**Comment by steveschmitt** [comment]
@twastvedt I've added a fix that may help - the code was converting back and forth between cases when generating the imports (table case -> model case -> file case) and that may have caused problems.  Fix will be in the next release.

The other thing to consider is that, we get the table names from INFORMATION_SCHEMA.TABLES, and the relationship information from INFORMATION_SCHEMA.KEY_COLUMN_USAGE.  If the casing of the table names don't match between those two sources for some reason, that may cause problems.

**Comment by twastvedt** [comment]
@steveschmitt Great, ok, thanks! Sorry to not get back with details. For the record, this was on MySql 5.7. I'll do some more testing, look at those table names, and give the next release a go when it comes out. Thanks!

**Comment by steveschmitt** [comment]
Fixed (I think) in 0.7.8

**Comment by twastvedt** [comment]
@steveschmitt Yes! Thanks!
